torch/_dynamo/eval_frame.py

```diff
@@ -151,6 +151,12 @@ def _set_stance(stance: DynamoStance) -> DynamoStance:
 
 
 def _callback_from_stance(callback):
+    if _stance.stance == "force_eager":
```

Comment:
I have no idea about this area of the code. Maybe fill me in on what we're doing here exactly? Or maybe ezyang should review lol.